### ISSUE TITLE
interactive graphs

### ISSUE THREAD

**Issue by davidak** [repology/repology#598]
Use some HTML5 canvas plotting library instead of images.

Has some advantages, like disabling graphs, show data when hover over datapoint, no need to render images, ...

I like [flot](http://www.flotcharts.org/), but the last release is from 2014. There is probably another library that is actively maintained.

**Comment by AMDmi3** [comment]
No. The site must be fully usable without javascript. Enhancing graphs with JS is a possibility though, like #386
